#!/usr/bin/env python
PACKAGE = "yolov8_result"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("distance_ratio_x", double_t, 0, "x scale-coordinate of specified center",    .5, 0,   1)
gen.add("distance_ratio_y", double_t, 0, "y scale-coordinate of specified center",    .5, 0,   1)

exit(gen.generate(PACKAGE, "yolov8_result", "Yolov8"))